boxoffice/templates/invoice_form.html

```diff
@@ -0,0 +1,102 @@
+                <label class="group-label">Street</label>
+                {{#errormsg.street_address}}<p class="form-error-msg">{{ errormsg.street_address }}</p>{{/}}
+              </div>
+              <div class="group inline-group left-group">
+                <input class="group-input {{#city}}filled{{/}}" type="text" name="city" value="{{city}}">
+                <span class="bar"></span>
+                <label class="group-label">City</label>
+                {{#errormsg.city}}<p class="form-error-msg">{{ errormsg.city }}</p>{{/}}
+              </div>
+              <div class="group inline-group right-group">
+                <input class="group-input {{#state}}filled{{/}}" type="text" name="state" value="{{state}}">
+                <span class="bar"></span>
+                <label class="group-label">State</label>
+                {{#errormsg.state}}<p class="form-error-msg">{{ errormsg.state }}</p>{{/}}
+              </div>
+              <div class="group inline-group left-group">
+                <input class="group-input {{#country}}filled{{/}}" type="text" name="country" value="{{country}}">
+                <span class="bar"></span>
+                <label class="group-label">Country</label>
+                {{#errormsg.country}}<p class="form-error-msg">{{ errormsg.country }}</p>{{/}}
+              </div>
+              <div class="group inline-group right-group">
+                <input class="group-input right-group {{#postcode}}filled{{/}}" type="text" name="postcode" value="{{postcode}}">
+                <span class="bar"></span>
+                <label class="group-label">Postcode</label>
+                {{#errormsg.postcode}}<p class="form-error-msg">{{ errormsg.postcode }}</p>{{/}}
+              </div>
+              <div class="assign-btn-wrapper">
+                <button type="submit" class="boxoffice-button boxoffice-button-action" on-click="submitInvoiceDetails(event)" {{#submittingInvoiceDetails}}disabled{{/}}>
+                  Submit {{#submittingInvoiceDetails}}<i class="fa fa-spinner fa-spin" intro='fly:{"x":0,"y":"0"}'>{{/}}
+                </button>
+              </div>
+            </form>
+            {{/}}
+            {{#errorMsg}}
+              <p class="stage-error-msg">{{ errorMsg }}</p>
+            {{/}}
+          {{else}}
+            <p class="confirmation-msg"><i class="fa fa-check"></i>Thank you for submitting the details, we will mail you the invoice shortly.</p>
+          {{/if}}
+          </div>
+        </div>
+      </script>
+    {%- endraw %}
+  </div>
+{% endblock %}
+{% block footerscripts %}
+  <script src='/static/js/views/invoice.js'></script>
+  <script type="text/javascript">
+    $(document).ready(function () {
+      Ractive.DEBUG = false;
+      var invoice = {};
+      invoice.id = "{{ order.id }}";
+      invoice.invoicee_name = "{{ order.buyer_fullname }}";
+      invoice.street_address = "F1, Malnad castle";
```

Comment:
These should be placeholders, not default text. When I begin to enter my details, I shouldn't have to remove existing details first.